scripts/qesap/test/unit/test_qesap_lib_cmd_ansible.py

```diff
@@ -82,12 +80,14 @@ def test_cmd_ansible(
         ["This is the ansible output", "Two lines of that"],
     )
 
-    calls = [mock.call(cmd=ansible_exe_call(inventory))]
-    calls.append(mock.call(cmd=ansible_sudo_wait_call(inventory)))
-    calls.append(mock_call_ansibleplaybook(inventory, playbook[0]))
-
     ret = cmd_ansible(data, tmpdir, False, False)
 
     assert ret == 0
 
-    subprocess_run.assert_has_calls(calls)
+    actual_calls = subprocess_run.call_args_list
+    expected_calls = ansible_regexp_calls(playbook[0])
+
+    assert len(actual_calls) == len(expected_calls)
+
+    for actual, expected in zip(actual_calls, expected_calls):
```

Comment:
I did it only here as POC, but we can use it in other tests and eventually remove  these 2 fixture at all
```
ansible_exe_call,
    ansible_sudo_wait_call,
    mock_call_ansibleplaybook,
```